spring-web/src/test/java/org/springframework/http/HttpHeadersTests.java

```diff
@@ -370,7 +370,7 @@ void expiresLong() {
 
 	@Test
 	void expiresZonedDateTime() {
-		ZonedDateTime zonedDateTime = ZonedDateTime.of(2008, 12, 18, 10, 20, 0, 0, ZoneId.of("GMT"));
+		ZonedDateTime zonedDateTime = ZonedDateTime.of(2008, 12, 18, 10, 20, 0, 0, ZoneOffset.UTC);
```

Comment:
this seems strange.


```
Changes have been made to spring-web/src/test/java/org/springframework/http/HttpHeadersTests.java by:
    tech.picnic.errorprone.refasterrules.TimeRulesRecipes
        tech.picnic.errorprone.refasterrules.TimeRulesRecipes$UtcConstantRecipe
```



kindly requesting you eye on this. Is this something to considering to open a issue for? Let me know and i will provide a ticket. Thanks @rickie

Reply:
It is actually behavior preserving and something we within Picnic do like to have consistent. I can see how it can be perceived slightly more opinionated but it's strictly speaking not incorrect. What would you like to open an issue for?